webapps-schema/src/main/java/io/camunda/webapps/schema/entities/usermanagement/EntityJoinRelation.java

```diff
@@ -20,12 +20,16 @@ public EntityJoinRelationFactory(
       this.child = child;
     }
 
-    public EntityJoinRelation createParent() {
-      return new EntityJoinRelation(parent.getType(), null);
+    public EntityJoinRelation<?> createParent() {
+      return new EntityJoinRelation<>(parent.getType(), null);
     }
 
-    public EntityJoinRelation createChild(final long parentKey) {
-      return new EntityJoinRelation(child.getType(), parentKey);
+    public EntityJoinRelation<Long> createChild(final long parentKey) {
```

Comment:
We could add a type parameter on the factory too and then only have one `createChild()` method. I know it's only needed temporarily, but it avoids creating an EntityJoinRelation of the wrong kind accidentally.